bear-language-requirements.yaml

```diff
@@ -0,0 +1,411 @@
+AlexBear:
+  - Natural Language
+AnnotationBear:
+ApertiumLintBear:
+  - Apertium
+ArtisticStyleBear:
+  - C
+  - C#
+  - C++
+  - Java
+  - Objective-C
+BanditBear:
+  - Python
+  - Python 2
+  - Python 3
+BashateBear:
+  - bash
+BootLintBear:
+  - HTML
+CISkipInspectBear:
+  - Git
+CMakeLintBear:
+  - CMake
+CPDBear:
+  - C#
+  - CPP
+  - Fortran
+  - Go
+  - JSP
+  - Java
+  - JavaScript
+  - Matlab
+  - Objective-C
+  - Octave
+  - PHP
+  - PLSQL
+  - Python
+  - Python 2
+  - Python 3
+  - Ruby
+  - Scala
+  - Swift
+CPPCheckBear:
+  - C
+  - C++
+CPPCleanBear:
+  - C++
+CPPLintBear:
+  - C++
+CSSAutoPrefixBear:
+  - CSS
+CSSCombBear:
+  - CSS
+CSSLintBear:
+  - CSS
+CSVLintBear:
+  - CSV
+CSecurityBear:
+  - C
+  - C++
+CSharpLintBear:
+  - C#
+CheckstyleBear:
+  - Java
+ClangASTPrintBear:
+  - C
+  - C++
+  - CUDA
+  - Objective-C
+  - Objective-C++
+  - OpenCL
+  - OpenMP
+ClangBear:
+  - C
+  - C++
+  - CUDA
+  - Objective-C
+  - Objective-C++
+  - OpenCL
+  - OpenMP
+ClangCloneDetectionBear:
+  - C
+  - C++
+  - CUDA
+  - Objective-C
+  - Objective-C++
+  - OpenCL
+  - OpenMP
+ClangComplexityBear:
+  - C
+  - C++
+  - CUDA
+  - Objective-C
+  - Objective-C++
+  - OpenCL
+  - OpenMP
+ClangFunctionDifferenceBear:
+  - C
+  - C++
+  - CUDA
+  - Objective-C
+  - Objective-C++
+  - OpenCL
+  - OpenMP
+CoffeeLintBear:
+  - CoffeeScript
+DartLintBear:
+  - Dart
+DennisBear:
+  - po
+  - pot
+DocGrammarBear:
+  - c
+  - cpp
+  - cs
+  - default
```

Comment:
Interesting ... `default` is not a language.  Maybe this is a bug in the bear, or this PR needs to create a special case to skip that one.